src/System.Net.Security/src/System/Net/Windows/SSPISecureChannelType.cs

```diff
@@ -0,0 +1,790 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System.ComponentModel;
+using System.Globalization;
+using System.Net.Security;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Principal;
+
+namespace System.Net
+{
+    // For SSL connections:
+    internal partial class SSPISecureChannelType : SSPIInterface
+    {
+        public const string MSSecurityPackage = "Microsoft Unified Security Protocol Provider";
+
+        public const Interop.Secur32.ContextFlags RequiredFlags = Interop.Secur32.ContextFlags.ReplayDetect |  Interop.Secur32.ContextFlags.SequenceDetect |
+                                                                  Interop.Secur32.ContextFlags.Confidentiality | Interop.Secur32.ContextFlags.AllocateMemory;
+
+        public const Interop.Secur32.ContextFlags ServerRequiredFlags =  RequiredFlags | Interop.Secur32.ContextFlags.AcceptStream;
+
+        private static volatile SecurityPackageInfoClass[] securityPackages;   
+
+        public void VerifyPackageInfo()
+        {
+            bool found = false;
```

Comment:
Breaking: Please change this back to the original code.
Please _do not_ change existing code during PAL work unless necessary.

```C#
     internal static SecurityPackageInfoClass GetVerifyPackageInfo(SSPIInterface secModule, string packageName, bool throwIfMissing)
        {
            SecurityPackageInfoClass[] supportedSecurityPackages = EnumerateSecurityPackages(secModule);
            if (supportedSecurityPackages != null)
            {
                for (int i = 0; i < supportedSecurityPackages.Length; i++)
                {
                    if (string.Compare(supportedSecurityPackages[i].Name, packageName, StringComparison.OrdinalIgnoreCase) == 0)
                    {
                        return supportedSecurityPackages[i];
                    }
                }
            }

            if (Logging.On)
            {
                Logging.PrintInfo(Logging.Web, SR.Format(SR.net_log_sspi_security_package_not_found, packageName));
            }

            if (throwIfMissing)
            {
                throw new NotSupportedException(SR.net_securitypackagesupport);
            }

            return null;
        }
```